### PR TITLE
dell: block owned TPM updates (#339)

### PR DESCRIPTION
**Pull request by superm1**
I confirmed with TPM team that this should have been blocking owned TPM upgrade too.  Sorry about that, I'll backport this to stable branches too.

### CODE REVIEW

**Comment by superm1** [comment]
The RPM build failing looks unrelated.  Those are either failing due to a fedora update or due to https://blog.travis-ci.com/2017-12-12-new-trusty-images-q4-launch

File: plugins/dell/fu-plugin-dell.c [hunk 1]
```diff
@@ -729,12 +729,16 @@ fu_plugin_dell_detect_tpm (FuPlugin *plugin, GError **error)
 	fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_INTERNAL);
 	fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_REQUIRE_AC);
 	fu_device_add_icon (dev, "computer");
-	if (out->flashes_left > 0) {
+	if (!((out->status) & TPM_OWN_MASK) &&
```
Comment (hughsie) [inline plugins/dell/fu-plugin-dell.c]: Wouldn't `if (out->status & TPM_OWN_MASK == 0)` be clearer? I'm not sure whether `TPM_OWN_MASK` means "can own" or "is owned".

Comment (superm1) [reply plugins/dell/fu-plugin-dell.c]: It means "is owned".  I'll make the logic clearer.

**Comment by superm1** [comment]
Adjusted, that look better?

**Comment by hughsie** [comment]
Lgtm, thx.

**Pull request merged**